### ISSUE TITLE
Update Lithuanian translations

### ISSUE THREAD

**Issue by lfx** [squidfunk/mkdocs-material#5250]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "lt",
  "action.edit": "Redaguoti šį puslapį",
  "action.skip": "Pereiti prie turinio",
  "action.view": "Žiūrėti puslapio šaltinius ⬅️",
  "announce.dismiss": "Daugiau neberodyti ⬅️",
  "blog.archive": "Archyvas ⬅️",
  "blog.categories": "Kategorijos ⬅️",
  "blog.categories.in": "į ⬅️",
  "blog.continue": "Skaityti toliau ⬅️",
  "blog.draft": "Ruošinys ⬅️",
  "blog.index": "Grįžti į indeksą ⬅️",
  "blog.meta": "Meta duomenys ⬅️",
  "blog.references": "Susieja saitai ⬅️",
  "clipboard.copy": "Kopijuoti į iškarpinę",
  "clipboard.copied": "Nukopijuota į iškarpinę",
  "consent.accept": "Sutikti ⬅️",
  "consent.manage": "Redaguoti nustatymus ⬅️",
  "consent.reject": "Atmesti ⬅️",
  "footer": "Poraštė",
  "footer.next": "Sekantis",
  "footer.previous": "Ankstesnis",
  "header": "Antraštė",
  "meta.comments": "Komentarai",
  "meta.source": "Išeitinis kodas",
  "nav": "Navigacija",
  "readtime.one": "1 min skaitymo ⬅️",
  "readtime.other": "# min skaitymo ⬅️",
  "rss.created": "RSS šaltinis ⬅️",
  "rss.updated": "RSS šaltinis atnaujinimams ⬅️",
  "search": "Paieška",
  "search.placeholder": "Paieška",
  "search.share": "Dalintis",
  "search.reset": "Išvalyti",
  "search.result.initializer": "Paieškos inicijavimas",
  "search.result.placeholder": "Įveskite norėdami pradėti paiešką",
  "search.result.none": "Atitinkančių dokumentų nerasta",
  "search.result.one": "1 atitinkantis dokumentas",
  "search.result.other": "# atitinkantys dokumentai",
  "search.result.more.one": "Dar 1 šiame puslapyje",
  "search.result.more.other": "Dar # šiame puslapyje",
  "search.result.term.missing": "Nerasta",
  "select.language": "Pasirinkti kalbą",
  "select.version": "Pasrinkti versiją",
  "source": "Eiti į saugyklą",
  "source.file.contributors": "Dalininkai ⬅️",
  "source.file.date.created": "Sukurta",
  "source.file.date.updated": "Paskutinis atnaujinimas",
  "tabs": "Skirtukai",
  "toc": "Turinys",
  "top": "Grįžti į viršų"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 33aeeaa00.

**Comment by squidfunk** [comment]
Released as part of [9.1.5](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.5).
